tools/ci/Dockerfile

```diff
@@ -20,7 +20,11 @@ RUN git clone https://github.com/FISCO-BCOS/FISCO-BCOS.git && cd FISCO-BCOS \
 FROM ubuntu:18.04
 LABEL maintainer service@fisco.com.cn
 
-RUN apt-get -q update && apt-get install -qy --no-install-recommends libssl-dev zlib1g-dev && rm -rf /var/lib/apt/lists/*
+RUN apt-get -q update && apt-get install -qy --no-install-recommends libssl-dev zlib1g-dev \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5424650)